source/types/globals.d.ts

```diff
@@ -0,0 +1,36 @@
+type ArgumentTypes<TFunction> = TFunction extends (...args: infer TInferredArgs) => infer TInferredReturnType ? TInferredArgs : never;
```

Comment:
I usually create a types folder in my projects for common stuff like you had. The reason I created the folder now is because there is also a `browser.d.ts` file now. I'm not a maintainer of this code base so let me know what you prefer.

Reply:
This has been reverted since we're back to just global.d.ts now.